data/mods/Russ_Extra_Stuff/terrain.json

```diff
@@ -0,0 +1,70 @@
+[{
+        "type" : "terrain",
+        "id" : "t_honeybee_hive",
+        "name": "honeybee hive",
+        "symbol": "#",
+        "color": "yellow",
+        "move_cost": 0,
+       "harvest_season": "SPRING",
+        "harvestable": "honeycomb",
+         "transforms_into": "t_honeybee_hive_empty",
+        "examine_action": "harvest_tree_shrub",
+        "flags": ["FLAMMABLE_ASH", "NOITEM", "TRANSPARENT"],
+        "bash": {
```

Comment:
You should add an option to deconstruct it. Having to smash a hive to get its contents doesn't make much sense.

Check out furniture entries in data/json/furniture.json on how to make a deconstruction entry.